app/views/tracks.js

```diff
@@ -11,6 +11,106 @@ const TracksView = Mn.CompositeView.extend({
 
     childView: TrackView,
 
+    initialize() {
+        this.dragObj = {
+            'dragDown': false,
+            'model': undefined,
+            'newIndex': undefined
+        };
+        this.listenTo(this.collection, 'sync', this.bindDragEvents);
+    },
+
+    bindDragEvents() {
+        $(document).mousemove(this.drag.bind(this));
+        $(document).mouseup(this.dragEnd.bind(this));
+        this.$el.find('#track-list .drag').mousedown(this.dragStart.bind(this));
```

Comment:
You can use

```js
events: {
    'mousedown .drag': 'dragStart'
}
```

which is a better approach 'coz it uses standard backbone's events, and bind just one event (here you bind one listener to each `.drag` el).